homeassistant/components/opensensemap/sensor.py

```diff
@@ -0,0 +1,106 @@
+"""Sensor for openSenseMap."""
```

Comment:
Adding a new platform should be done in a follow-up to keep this PR as small as possible.